playground/mocks/config/responseConfig.js

```diff
@@ -503,6 +503,32 @@ const phoneEnroll = {
   ],
 };
 
+const duoMFAEnroll = {
+  '/idp/idx/introspect': [
+    'authenticator-enroll-select-authenticator',
+  ],
+  '/idp/idx/credential/enroll': [
+    'authenticator-enroll-duo',
+  ],
+  '/idp/idx/challenge/answer': [
+    'success',
+    //'error-authenticator-verification-duo',
+  ],
+};
+
+const duoMFAVerify = {
```

Comment:
@haishengwu-okta: We should probably should start thinking about breaking these mock sequences into multiple files. 😄

Reply:
@jmelberg-okta yes!
I was thinking a simple idea like `.widgetrc.js`, which isn't source controlled and developer is free to change.
We could have a file like `.playgroundrc.js` which export set of mock for testing.
Open for more ideas.

Reply:
created https://oktainc.atlassian.net/browse/OKTA-369483 to follow up